### ISSUE TITLE
z_dim for 512x512 model

### ISSUE THREAD

**Issue by SsnL** [ajbrock/BigGAN-PyTorch#16]
The paper mentions in Appendix B that 512x512 model should use 160 dimensional z. But both in this codebase and TFHub, a 128 dimensional z space is used (which is smaller than the 140 dimensional z space for 256x256 model).  Which is correct? If the code is correct, should the paper be updated, and why does it use a smaller z space than 256x256 model?

**Comment by SsnL** [comment]
@ajbrock any update on this?

**Comment by ajbrock** [comment]
Hi Tongzhou,

This looks like a typo or a mismatch somewhere between the paper and that particular model setup--we'll take a look and see about updating. FWIW the model is quite robust to the choice of z dimension (in the paper we mention that we are able to train with very low dimensionalities, on the order of 32 dims IIRC) so I wouldn't read too much into this; there's no explicit design choice to e.g. make the z-dim smaller for 512x512 than 256x256. BigGAN-Deep uses a 128-dim z for all resolutions, for example.

Andy
